src/core/Akka.TestKit/TestKitBase_Expect.cs

```diff
@@ -537,7 +528,7 @@ private async ValueTask InternalExpectNoMsgAsync(TimeSpan duration, Cancellation
         public T ExpectMsgAnyOf<T>(IEnumerable<T> messages, CancellationToken cancellationToken = default)
         {
             return ExpectMsgAnyOfAsync(messages, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.